types/transform-coordinates/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
this is disputable dependency, it just because `console` usage in tests IMO. Please remove this
to speed up CI tests and save CPU cycles, I'll add comments in tests